test/e2e/tests/resize_e2e_test.go

```diff
@@ -288,6 +290,17 @@ func validateRead(publishDir, testFileName, testFileContents string, instance *r
 	Expect(strings.TrimSpace(string(readContents))).To(Equal(testFileContents))
 }
 
+func validateVolumeStats(publishDir, volID string, tc *remote.TestContext) {
+	available, capacity, used, inodesFree, inodes, inodesUsed, err := tc.Client.NodeGetVolumeStats(volID, publishDir)
+	Expect(err).To(BeNil(), "Failed to get node volume stats: %v", err)
+	Expect(equalWithinEpsilon(available, minVolumeSize, defaultEpsilon)).To(BeTrue())
+	Expect(equalWithinEpsilon(capacity, minVolumeSize, defaultEpsilon)).To(BeTrue())
+	Expect(equalWithinEpsilon(used, 0, defaultEpsilon)).To(BeTrue())
```

Comment:
there have been file written to the volume, so here used should not be 0

Reply:
Yes, but it's hard to quantitive the usage. So let's set a threshold of 70G at this point, and modify it latter if things changed.

Reply:
the usage should be deterministic in this case right? it's just "test" with whatever overhead a file has. There's no reason to have 70G of epsilon here and the usage should not be 0 either.